src/view/com/post-thread/PostThreadItem.tsx

```diff
@@ -700,7 +723,7 @@ const useStyles = () => {
     },
     expandedInfo: {
       flexDirection: 'row',
-      padding: 10,
+      paddingVertical: 10,
```

Comment:
There's a lot of stuff in an expanded post item, and it looked better all left-aligned. However, we may want to adjust how we display all this bc it's kind of busy.